File: ioncwalletsdk/src/main/java/org/ionc/wallet/sdk/callback/OnTxRecoderCallback.java
package org.ionc.wallet.sdk.callback;

import org.ionc.wallet.sdk.bean.TxRecoderBean;

import java.util.ArrayList;

/**
 * user: binny
 * date:2018/12/7
 * description：交易记录
 */
public interface OnTxRecoderCallback {
    /**
     * 请求成功
     */
    void onTxRecoderSuccess(ArrayList<TxRecoderBean.DataBean.ItemBean> beans);


    /**
     * 请求失败
     */
    void onTxRecoderFailure(String error);
}

File: app/src/main/java/org/ionchain/wallet/mvp/callback/OnTxRecoderCallback.kt
package org.ionchain.wallet.mvp.callback

import org.ionchain.wallet.bean.TxRecoderBean

/**
 *
 * AUTHOR binny
 *
 * TIME 2018/11/13 16:33
 *
 *
 */
interface OnTxRecoderCallback:OnLoadingView {

    /**
     * 请求成功
     * */
    fun onTxRecoderSuccess(beans: ArrayList<TxRecoderBean.DataBean.ItemBean>)


    /**
     * 请求失败
     * */
    fun onTxRecoderFailure(error: String)
}